### PR TITLE
chore(docs): add provider schema

### PR DESCRIPTION
**Pull request by johnstcn**
Adds some words to the provider schema.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,12 +3,19 @@
...
+  The Envbuilder provider can be used to check for the presence of a Docker image previously built by Envbuilder https://github.com/coder/envbuilder.
+  This allows re-using a previously built image pushed to a Docker registry without having to rebuild it.
+  An image built by Envbuilder also embeds the Envbuilder binary inside the image, allowing for the image to be used to build other images.
 ---
 
 # envbuilder Provider
 
+# Envbuilder Provider
 
+The Envbuilder provider can be used to check for the presence of a Docker image previously built by [Envbuilder](https://github.com/coder/envbuilder).
```
Comment (mafredri) [inline docs/index.md:16]:
```suggestion
The Envbuilder provider can be used to check for the presence of a container image previously built by [Envbuilder](https://github.com/coder/envbuilder).
```

Should we be more generic?

File: docs/index.md [hunk 2]
```diff
@@ -3,12 +3,19 @@
...
+This allows re-using a previously built image pushed to a Docker registry without having to rebuild it.
```
Comment (mafredri) [inline docs/index.md:17]:
```suggestion
This allows re-using a previously built image pushed to a container registry without having to rebuild it.
```

File: docs/index.md [hunk 3]
```diff
@@ -3,12 +3,19 @@
...
+An image built by Envbuilder also embeds the Envbuilder binary inside the image, allowing for the image to be used to build other images.
```
Comment (mafredri) [inline docs/index.md:18]: This is a bit of a fringe case, do we want to mention it, might seem like a feature and maybe it is, but we'd be making it official? 😄

**Review by mafredri — APPROVED** [review]

File: docs/index.md (hunk 3 above)
Comment (johnstcn) [reply docs/index.md:18]: Fair :+1:

File: docs/index.md [hunk 4]
```diff
@@ -3,12 +3,17 @@
...
 description: |-
-  
+  Envbuilder Provider
+  The Envbuilder provider can be used to check for the presence of a container image previously built by Envbuilder https://github.com/coder/envbuilder.
+  This allows re-using a previously built image pushed to a container registry without having to rebuild it.
 ---
 
 # envbuilder Provider
 
+# Envbuilder Provider
```
Comment (matifali) [inline docs/index.md:13]: duplicate?

**Pull request merged**